suite-native/state/src/extraDependencies.ts

```diff
@@ -42,7 +42,7 @@ export const extraDependencies: ExtraDependencies = mergeDeepObject(extraDepende
             params: undefined,
         }),
         selectThpSettings: state => ({
-            hostName: Device.deviceName,
+            hostName: Platform.OS === 'ios' ? Device.modelName : Device.deviceName,
```

Comment:
Might be helpful just commenting that this is due to some iOS capabilities, but it's visible in docs, so not a biggie.